src/main/java/com/epam/learn/springcore/entity/Trainee.java

```diff
@@ -1,22 +1,30 @@
 package com.epam.learn.springcore.entity;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.*;
 import lombok.*;
-import lombok.experimental.SuperBuilder;
-import lombok.extern.jackson.Jacksonized;
 
-import java.util.Date;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Set;
 
 @Getter
 @Setter
-@SuperBuilder
-@Jacksonized
-@ToString(callSuper = true)
-@NoArgsConstructor
-@AllArgsConstructor
-public class Trainee extends User {
-    @JsonProperty
-    private Date dateOfBirth;
-    @JsonProperty
+@Entity
```

Comment:
add @Builder for better construction of class

Reply:
Ok